plugins/catalog-backend-module-msgraph/src/microsoftGraph/client.ts

```diff
@@ -214,7 +214,7 @@ export class MicrosoftGraphClient {
   ): Promise<Response> {
     // Make sure that we always have a valid access token (might be cached)
     const token = await this.tokenCredential.getToken(
-      'https://graph.microsoft.com/.default',
+      `${this.baseUrl}/.default`,
```

Comment:
According to the config schema the base URL might be something like `https://graph.microsoft.com/v1.0`, so I'm assuming there's a high risk that this'll break for the regular use-case?

Do you know if we can reliably find the correct base URL for this request, or do we perhaps need a new configuration parameter?

Reply:
Yeah, I was thinking the same thing and actually had URL parsing and using the protocol + host here instead of the baseUrl, BUT... Accessing https://graph.microsoft.com/.default doesn't even look like the right location but instead one should actually use the https://graph.microsoft.com/v1.0/.default

Reply:
As you can also see, it requires the version number in the URL when accessing the https://graph.microsoft.com/.default

```json
{"error":{"code":"BadRequest","message":"Invalid version.","innerError":{"date":"2023-01-27T10:48:29","request-id":"5905f7bc-3339-4a37-8d79-53770f7fa96d","client-request-id":"5905f7bc-3339-4a37-8d79-53770f7fa96d"}}}
```

Reply:
Ah alright, might be a backwards compatibility thing then